### ISSUE TITLE
SimCLR for audio classification

### ISSUE THREAD

**Issue by FirasHm** [thunderInfy/simclr#2]
Hello, thank you for this great work!
I am using simclr for audio (speech) processing, by extracting features as images from the audio clips and then use them to train/fine-tune a model. My first experiments provides me 28% (accuracy) for 150 epochs. Here what I did:

1- Extract MFCCs spectrogram from speech audio
2- Fine tune simclr by training the linear classifier

Is it the right way to use simclr ? Do you think that 28% is a "normal" accuracy? I think it is due to the fact that MFCCs images are very similar that the model can not classify them.

Thanks in advance
